src/content/reference/release-versions/_index.md

```diff
@@ -1,58 +1,118 @@
 
-We generally try to increment our Major version only when there is a new Kubernetes release.
+### Major version {#major}
 
-| Giant Swarm release version | Kubernetes version | Status |
-|:-----------:|:------------:|:-----:|
-| **9.x.x** | 1.15.x\* | Available |
-| **11.x.x** | 1.16.x | Available |
-| **12.x.x** | 1.17.x | Available |
-| **13.x.x** | 1.18.x | Available soon |
+The Kubernetes project providing the most important component of our tenant clusters. So we align our release versioning with the Kubernetes releases.
 
-\*) On AWS, **v9.2.0** and **v9.3.x** and up includes **Kubernetes 1.16.x**.
+**Convention**: For each new _minor_ Kubernetes release we provide a new _major_ release.
 
-## Versions that support node pools
+The following table shows which of our major version contain which Kubernetes release.
 
-- AWS **{{% first_aws_nodepools_version %}}** and up.
+| Giant Swarm release version | Kubernetes version | Availability |
+|:---------------------------:|:------------------:|:------------:|
+| **9.x.x**                   | 1.15.x\*           | Available    |
+| **11.x.x**                  | 1.16.x             | Available    |
+| **12.x.x**                  | 1.17.x             | Available    |
+| **13.x.x**                  | 1.18.x             | ETA November 2020 |
+| **14.x.x**                  | 1.19.x             | ETA December 2020 |
 
-- Azure **{{% first_azure_nodepools_version %}}** and up.
+\*) As an exception from the convention, on AWS, releases v9.2.0 and v9.3.x include Kubernetes v1.16.x.
 
-## Versions with an optional ingress controller
+We test every new major release, bringing a new Kubernetes minor release, against the CNCF [conformance test suite](https://github.com/cncf/k8s-conformance).
+Every release, from patch to major, also undergoes automated integration testing.
+
+A major release may contain more important changes, apart from a new Kubernetes release. Check the [Details about releases and features](#release-details) section for more information.
+
+### Minor version {#minor}
+
+We increase (bump) the minor version number when a release adds new functionality, while maintaining the functionality of the stack as it was in the previous version. This can include minor upgrades of third party components, except for Kubernetes.
+
+### Patch version {#patch}
+
+We use patch releases to publish bug fixes, security fixes, or to make changes to the observability while maintaining the given functionality of the stack. This can include any sort of patch upgrades of third party components.
+
+### Release lifecycle {#lifecycle}
+
+We maintain **two different major releases** at the same time, which means that you have two different Kubernetes minor versions to chose from.
+
+Whenever a new Kubernetes minor version is released by the Kubernetes project, we aim to make that version available in a new major release within 30 days.
```

Comment:
This phrase could be extended to all components of the stack, so we highlight our duty of maintaining and upgrading the whole Cloud Native stack :)